blivet/osinstall.py

```diff
@@ -1188,6 +1184,165 @@ def write(self):
         Blivet.write(self)
 
         self.make_mtab()
+        self.fsset.write()
+
+    def mountpoints(self):
+        return self.fsset.mountpoints
```

Comment:
Wait -- is this defined twice here? Maybe you meant to keep this one since it uses the fsset?